doc/api/process.md

```diff
@@ -1503,6 +1503,17 @@ tarball.
   - `'Argon'` for the 4.x LTS line beginning with 4.2.0.
   - `'Boron'` for the 6.x LTS line beginning with 6.9.0.
   - `'Carbon'` for the 8.x LTS line beginning with 8.9.1.
+* `majorVersion` {number} The major version of Node.js.
+* `minorVersion` {number} The minor version of Node.js.
+* `patchVersion` {number} The patch version of Node.js.
+* `prereleaseTag` {string} The SemVer pre-release tag for Node.js.
+* `computedVersion` {number} The result of
+  `(majorVersion << 16) + (minorVersion << 8) + patchVersion`
+
+  This property is useful for inline version checks. Using the process above a
```

Comment:
I think this entire description is unnecessary it should be pretty obvious how the version properties could be used.